python/pywatchman/__init__.py

```diff
@@ -553,7 +553,7 @@ def receive(self):
         if not buf[0]:
             raise WatchmanError('empty watchman response')
 
-        elen, recv_bser_version, recv_bser_capabilities = bser.pdu_info(buf[0])
+        recv_bser_version, recv_bser_capabilities, elen = bser.pdu_info(buf[0])
```

Comment:
This order seems more reasonable to me (this is the order they are present in the header).